app/models/participant_outcome/npq.rb

```diff
@@ -61,6 +63,23 @@ def passed
     def not_passed
       where.not(state: :passed)
     end
+
+    def latest_per_declaration
```

Comment:
Is the intention of this method to check the outcome in the table is the latest? I wonder if we can just just `declaration.outcomes.latest` is equal to the current outcome?